cordova-lib/src/cordova/plugin.js

```diff
@@ -284,6 +284,11 @@ module.exports = function plugin(command, targets, opts) {
                                 }
                             }
                         }
+                    }).then(function(){
```

Comment:
Nit: Consider adding a space after ')' - aids readability. This is an issue in most of the code below.